src/coreclr/jit/importer.cpp

```diff
@@ -11308,16 +11308,19 @@ GenTree* Compiler::impAssignMultiRegTypeToVar(GenTree*             op,
 {
     unsigned tmpNum = lvaGrabTemp(true DEBUGARG("Return value temp for multireg return"));
     impAssignTempGen(tmpNum, op, hClass, CHECK_SPILL_ALL);
-    GenTree* ret = gtNewLclvNode(tmpNum, lvaTable[tmpNum].lvType);
+
+    LclVarDsc* varDsc = lvaGetDesc(tmpNum);
+
+    // The following is to exclude the fields of the local to have SSA.
```

Comment:
```suggestion
    // Set "lvIsMultiRegRet" to block promotion under "!lvaEnregMultiRegVars".
```
(The SSA bit is no longer true)